src/OSPSuite.Presentation/Presenters/Charts/ChartDisplayPresenter.cs

```diff
@@ -539,7 +554,7 @@ private void bindChartToView()
          if (Chart.PreviewSettings)
             setDisplay(areChartWidthAndHeightDefined ? DockStyle.None : DockStyle.Fill, Chart.FontAndSize, showingPreview: true);
          else
-            setDisplay(DockStyle.Fill, ChartFontAndSizeSettings.Default, showingPreview: false);
```

Comment:
Setting previewSettings to true was the only way before to custom display chart. (Feedback title needs to be smaller it seems)
Now this is a parameter of the Edit method